src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(num) {
+    if (num === undefined) {
```

Comment:
The check for `num === undefined` is correct for determining when to return the accumulated sum. However, ensure that this logic aligns with the task requirements, which may specify how to handle non-numeric inputs or other edge cases.